qwerty_mlir/include/CCirc/IR/QCircDialect.h

```diff
@@ -0,0 +1,19 @@
+//===- QCircDialect.h - QCirc dialect -----------------*- C++ -*-===//
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef DIALECT_INCLUDE_QCIRC_IR_QCIRC_DIALECT_H
+#define DIALECT_INCLUDE_QCIRC_IR_QCIRC_DIALECT_H
+
+#include "mlir/IR/Dialect.h"
+
+#include "QCirc/IR/QCircOpsDialect.h.inc"
+
+namespace qcirc {
+
+// Needed for the unfortunate LLVMConstantArrayOp (see QCircOps.td)
+void registerQCircDialectTranslation(mlir::DialectRegistry &registry);
+
+} // namespace qcirc
```

Comment:
Please obliterate these lines. They are raising my blood pressure just looking at them (they are a past sin of mine)